pom.xml

```diff
@@ -288,27 +311,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
```

Comment:
As we're switching from source/target to release, there's no need for animal sniffer anymore